Specs/DateTime/Japanese/TimeExtractor.json

```diff
@@ -493,22 +455,8 @@
       }
     ]
   },
-  {
-    "Input": "日中",
```

Comment:
Please make sure we have this in the time range specs. Thanks.

Reply:
added